client/components/marketing-survey/gsuite-cancel-purchase-dialog/gsuite-cancellation-features.jsx

```diff
@@ -17,29 +17,43 @@ class GSuiteCancellationFeatures extends Component {
 		this.props.recordTracksEvent( 'calypso_purchases_gsuite_remove_purchase_learn_more_click' );
 	};
 
+	getTimePeriod = () => {
+		const { status, translate } = this.props;
+		return status === 'suspended'
+			? translate( 'immediately' )
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **37** times:
`translate( '  Immediately' )` *ES Score: 11*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/update-cancel-google-workspace-survey)